forest/src/cli/chain_cmd.rs

```diff
@@ -82,18 +85,28 @@ impl ChainCommands {
 
                 let epoch = tipset.unwrap_or(chain_head.epoch());
 
+                let now = OffsetDateTime::now_utc();
+
+                let month_string = format!("{:02}", now.month() as u8);
+                let year = now.year();
+                let day = now.day();
+                let chain_name = cfg.chain.name.clone();
+
                 let output_path = match output_path {
-                    Some(path) => path.to_owned(),
-                    None => {
-                        let now = OffsetDateTime::now_utc();
-                        format!(
-                            "forest_snapshot_{}-{}-{}_height_{}.car",
-                            now.year(),
-                            now.month(),
-                            now.day(),
-                            epoch,
-                        )
+                    Some(path) => {
+                        let mut vars = HashMap::new();
+                        vars.insert("year".to_string(), year.to_string());
+                        vars.insert("month".to_string(), month_string);
+                        vars.insert("day".to_string(), day.to_string());
+                        vars.insert("chain".to_string(), chain_name);
+                        vars.insert("heigth".to_string(), epoch.to_string());
+                        // we need to unwrap here, as in case the string is malformed one needs to stop the execution
+                        strfmt(path, &vars).unwrap()
```

Comment:
`unwrap` is not appropriate here. These errors may be shown to the user and should therefore be formatted properly.